docker-compose-phpunit.yml

```diff
@@ -0,0 +1,23 @@
+version: "3"
```

Comment:
I wonder if we should tuck files like this away under the docker sub directory just to keep from cluttering the root dir?

Reply:
Sounds good, now done in 7d912f7